### ISSUE TITLE
Embedded mode no longer works with NextJS 12

### ISSUE THREAD

**Issue by rburgst** [keystonejs/keystone#6847]
# Bug report

Please use this form for bug reports only. If you have a question, feature request or want to chat with us, please select the appropriate option here - https://github.com/keystonejs/keystone/issues/new/choose

## Describe the bug

Embedded mode no longer works with nextjs 12.

## To Reproduce

Steps to reproduce the behaviour. Please provide code snippets or a repository:

1. Go to https://keystonejs.com/docs/walkthroughs/embedded-mode-with-sqlite-nextjs
2. Follow the instructions
3. run `yarn dev`
4. the server fails to boot with the error

```
yarn dev
yarn run v1.22.17
$ next dev
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
✨ Generated GraphQL and Prisma schemas
We detected TypeScript in your project and reconfigured your tsconfig.json file for you. Strict-mode is set to false by default.

The following suggested values were added to your tsconfig.json. These values can be changed to fit your project's needs:

	- incremental was set to true

Error: If you see this error, please open an issue with your Next.js version and @preconstruct/next version. The Next Babel loader could not be found
    at Object.nextConfig.webpack (/Users/rainer/git/private/turnen/turnen-next2/node_modules/@preconstruct/next/index.js:22:13)
    at Object.getBaseWebpackConfig [as default] (/Users/rainer/git/private/turnen/turnen-next2/node_modules/next/dist/build/webpack-config.js:1327:32)
    at async Promise.all (index 0)
    at async Span.traceAsyncFn (/Users/rainer/git/private/turnen/turnen-next2/node_modules/next/dist/trace/trace.js:74:20)
    at async Span.traceAsyncFn (/Users/rainer/git/private/turnen/turnen-next2/node_modules/next/dist/trace/trace.js:74:20)
    at async HotReloader.start (/Users/rainer/git/private/turnen/turnen-next2/node_modules/next/dist/server/dev/hot-reloader.js:321:25)
    at async DevServer.prepare (/Users/rainer/git/private/turnen/turnen-next2/node_modules/next/dist/server/dev/next-dev-server.js:289:9)
    at async /Users/rainer/git/private/turnen/turnen-next2/node_modules/next/dist/cli/next-dev.js:126:9
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

## Expected behaviour

The server should boot up properly.

## System information

- OS: macOS Big Sur
- next.js version 12.0.1
- keystone version 26.1.1

## Additional context

Package.json
```json
{
  "name": "turnen-next2",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "@keystone-next/auth": "^34.0.0",
    "@keystone-next/fields-document": "^11.0.0",
    "@keystone-next/keystone": "^26.1.1",
    "next": "12.0.1",
    "next-auth": "^3.29.0",
    "react": "17.0.2",
    "react-dom": "17.0.2"
  },
  "devDependencies": {
    "@types/node": "16.11.6",
    "@types/react": "17.0.33",
    "eslint": "7",
    "eslint-config-next": "12.0.1",
    "typescript": "4.4.4"
  }
}
```

**Comment by jzyds** [comment]
Same error

**Comment by bladey** [comment]
Hey @rburgst and @jzyds, we're aware of this, Next 12 won't work until we upgrade Keystone to Next 11, I'll let you know once this is done, hopefully in the next release - https://github.com/keystonejs/keystone/pull/6835

**Comment by JedWatson** [comment]
We just released `keystone-next/keystone@27.0.0` which includes Next 12
